mysql-test/main/rpl_mysqldump_slave.result

```diff
@@ -204,13 +201,11 @@ CHANGE MASTER TO MASTER_LOG_FILE='slave-bin.000001', MASTER_LOG_POS=BINLOG_START
 
 -- A corresponding to the above master-data CHANGE-MASTER settings to the slave gtid state is printed as comments later in the file.
 
---
--- The following is the SQL position of the replication taken from SHOW SLAVE STATUS at the time of backup.
--- Use this position when creating a clone of, or replacement server, from where the backup was taken.
+-- The following is the replication SQL position taken from SHOW SLAVE STATUS at the time of backup.
+-- Use this position when creating a clone or replacement server from where the backup was taken.
 -- This new server will connects to the same primary server(s).
 --
-
--- A corresponding to the below dump-slave CHANGE-MASTER settings to the slave gtid state is printed later in the file.
+-- The replica GTID position corresponding to the below CHANGE-MASTER settings is printed later in the file.
```

Comment:
* The previous line calls them *primaries* since before this edit, even though nowhere in the server supports the ‘primary’ term.
  This line might as well match by calling it a *replica*, then.
* This is a position, not a ‘state’.